Cabal/src/Distribution/Compat/Async.hs

```diff
@@ -150,12 +148,10 @@ data AsyncCancelled = AsyncCancelled
 
 {- FOURMOLU_DISABLE -}
```

Comment:
I think you can remove the Fourmolu directives when you're getting rid of the CPP.

Reply:
I can do that.  Now pushed!